source/code-generation/query-converter/convert-oracle-packages.txt

```diff
@@ -0,0 +1,145 @@
+.. _rm-convert-oracle-packages:
+
+=======================
+Convert Oracle Packages
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can import and convert your Oracle Packages to MongoDB code 
+with the query converter. The query converter considers the mapping 
+rules and schema transformations defined in your project when 
+converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-view-query-conversion-history.rst
+
+Before you Begin
+----------------
+
+- MongoDB does not have an official synonym object type for Oracle 
+  packages. Instead, MongoDB supports custom JavaScript and 
+  driver code that hosts the database logic. Part of your application 
+  modernization journey is choosing the programming language and hosting 
```

Comment:
(suggestion)
Would you mind checking around to see if there's something public that we can link to, describing the application modernization journey. This page came up in google search: https://www.mongodb.com/resources/solutions/use-cases/application-modernization-factory-datasheet

Also, "application modernization journey" sounds a bit jargony. I wonder if we could tweak a little to make it more straight forward? Maybe something like "the first step to ensuring that your application migrates efficiently to MongoDB, ..." Just spitballing.